mmdet/datasets/pipelines/transforms.py

```diff
@@ -1239,7 +1239,6 @@ class RandomCenterCropPad(object):
             Only available in train mode.
         border (int): max distance from center select area to image border.
             Only available in train mode.
-        mean (sequence): Mean values of 3 channels.
```

Comment:
Why delete this line?

Reply:
It is a mistake when merging master branch.